user-manual.md

```diff
@@ -196,102 +193,48 @@ Um die Eingabe so einfach wie möglich zu gestalten, werden zunächst persönlic
 
 Nach erfolgreicher Eingabe dieser Informationen werden eine oder mehrere Unterkunftsmöglichkeiten angezeigt. Aufgrund der Vielzahl an möglichen Eingaben werden alle acht Endpunkte nachfolgend in Beispielen aufgelistet.
```

Comment:
Hier steht dass alle acht Endpunkte in Beispielen aufgelistet sind, da sind aber nur drei Beispiele - hab ich da jetzt was übersehen?

Reply:
Sehe auch nur 3

Reply:
Das war noch ein überbleibsel von früher, dass da gar nicht hingehört.
Habs gelöscht.